test/cp.js

```diff
@@ -125,17 +125,23 @@ assert.equal(shell.cat('resources/cp/dir_a/z'), shell.cat('tmp/cp/dir_a/z')); //
 
 //recursive, creates dest dir since it's only one level deep (see Github issue #44)
 shell.rm('-rf', 'tmp/*');
-shell.cp('-r', 'resources/issue44/*', 'tmp/dir2');
+shell.cp('-r', 'resources/issue44', 'tmp/dir2');
```

Comment:
In \*nix, `cp -r resources/issue44/* tmp/dir2` is not valid. If there is only one file in `resources/issue44`, it will be copied into `tmp` and be renamed `dir2`. If there is more than one file in `resources/issue44`, the command will fail.

The intention here is to copy all of the files in `resources/issue44` to `tmp/dir2`, where `dir2` will be created. This is achieved with the new command (and in *nix as well).